keypoints/include/pcl/keypoints/sift_keypoint.h

```diff
@@ -107,13 +107,15 @@ namespace pcl
       using Keypoint<PointInT, PointOutT>::surface_;
       using Keypoint<PointInT, PointOutT>::tree_;
       using Keypoint<PointInT, PointOutT>::initCompute;    
+      using Keypoint<PointInT, PointOutT>::keypoints_indices_;
 
       /** \brief Empty constructor. */
       SIFTKeypoint () : min_scale_ (0.0), nr_octaves_ (0), nr_scales_per_octave_ (0), 
         min_contrast_ (-std::numeric_limits<float>::max ()), scale_idx_ (-1), 
         out_fields_ (), getFieldValue_ ()
       {
         name_ = "SIFTKeypoint";
+        keypoints_indices_ = PointIndices::Ptr(new pcl::PointIndices ());
```

Comment:
This is not necessary, `keypoints_indices_` is initialized before the computation starts by the base class [here](https://github.com/PointCloudLibrary/pcl/blob/master/keypoints/include/pcl/keypoints/impl/keypoint.hpp#L116).